### DISCUSSION TITLE
Set up dockerized AIO with a reverse proxy on a different host

### DISCUSSION THREAD

**Discussion by spyrosmich** [nextcloud/all-in-one#2265] [Questions] [answered]
Hello, in https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md it is clearly noted that "_The above configuration will only work if your reverse proxy is running directly on the host that is running the docker daemon_". 

As we already use an Apache proxy for our internal network, running on a different host (actually a VM guest on another host), is it possible to make AIO to work or I should forget about it and resort to setting up a VM nextcloud installation? 

Thanks in advance!

**Comment by szaimen** [comment, marked as answer]
Hi, if you read further down, you get your answer 😉

**Reply by spyrosmich** [reply]
Hi, thanks, after exhausting the time I had allocated to this task to no avail, I've resorted to installing NC in a vm and made it work with our Apache proxy. 

If you could find some time to make the documentation a little more explanatory (e.g. what APACHE_PORT needs to have if the proxy is on another host etc.), I'm sure it would be appreciated by many users. I had intended to contribute my solution to [this ticket](https://github.com/nextcloud/all-in-one/discussions/588) but, well, maybe next time :)

**Reply by szaimen** [reply]
> APACHE_PORT

The apache_port gets exposed on the host that runs the docker daemon. So choosing the port is up-to-you.
